catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/converter/TestFromIcebergSortOrder.java

```diff
@@ -96,6 +101,10 @@ public void testFromSortOrder() {
               ? NullOrdering.NULLS_FIRST
               : NullOrdering.NULLS_LAST,
           sortOrder.nullOrdering());
+      String icebergSortOrderString =
+          convertIcebergTransform(sortField) + idToName.get(sortField.sourceId()) + ")";
```

Comment:
could you place `convertIcebergTransform(sortField) + idToName.get(sortField.sourceId()) + ")";` in one function?

Reply:
updated.